agent/php_globals.h

```diff
@@ -75,6 +75,7 @@ typedef struct _nrphpglobals_t {
   int apache_threaded; /* 1 if a threaded MPM is in use, 0 otherwise */
   int preload_framework_library_detection; /* Enables preloading framework and
                                               library detection */
+  char* docker_id;
```

Comment:
Please add a description for this field.

Reply:
Added in fc0f631a710c509cb2935ce0c5a508ccea61694a